src/python_pybind11/CMakeLists.txt

```diff
@@ -1,4 +1,16 @@
-if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
+# Detect if we are doing a standalone build of the bindings, using an external gz-math
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  cmake_minimum_required(VERSION 3.22.1)
```

Comment:
The top level CMakeLists.txt file has a minimum version of 3.10.2. Is this necessary?  If we require 3.22.1 to be able to run this, we should document the requirement in installation.md

Reply:
the `CMAKE_REQUIRE_FIND_PACKAGE_pybind11` variable on line 10 [requires cmake 3.22](https://cmake.org/cmake/help/latest/variable/CMAKE_REQUIRE_FIND_PACKAGE_PackageName.html), so I used the same version as Ionic to simplify merges

I will update the documentation to indicate the required cmake version

Reply:
https://github.com/gazebosim/gz-math/pull/640/commits/2448824e9d804d800bf3745b209910cf919b5976